### ISSUE TITLE
Training data should be shuffled by default

### ISSUE THREAD

**Issue by dfalbel** [mlverse/luz#70]
When using a torch dataset or list, we need to make sure data is shuffled by default.
